numpy/distutils/ccompiler.py

```diff
@@ -443,14 +443,6 @@ def CCompiler_show_customization(self):
     Printing is only done if the distutils log threshold is < 2.
 
     """
-    if 0:
```

Comment:
It would be worth looking at the git blame to work out why this code was disabled but not removed.

Reply:
It was in the very [first version](https://github.com/numpy/numpy/commit/575d373479c63a42bc4a729a058da31a74e75d3e#diff-52c468e38e7278f69142ae2a02533908R161).

Reply:
Yes, this is the first version and should be deleted or commented.